arcgis-ios-sdk-samples/Edit data/Edit and sync features/README.md

```diff
@@ -0,0 +1,43 @@
+# Edit and sync features
+
+Synchronize offline edits with a feature service.
+
+![Edit and sync features](edit-and-sync-features.png)
+
+## Use case
+
+A survey worker who works in an area without an internet connection could take a geodatabase of survey features offline at their office, make edits and add new features to the offline geodatabase in the field, and sync the updates with the online feature service after returning to the office.
+
+## How to use the sample
+
+Pan and zoom to position the red rectangle around the area to be taken offline. Tap "Generate geodatabase" to take the area offline. To edit features, tap to select a feature, and tap again anywhere else on the map to move the selected feature to the clicked location. To sync the edits with the feature service, tap the "Sync geodatabase" button.
+
+## How it works
+
+1. Create a geodatabase sync task from a URL to a feature service.
+2. Generate the geodatabase sync task to create geodatabase parameters by passing in an envelope extent as the parameter.
+3. Create a geodatabase job, passing in the parameters and a path to where the geodatabase should be downloaded locally.
+4. Start the job and get a geodatabase as a result.
+5. Load the geodatabase and get its feature tables. Create feature layers from the feature tables and add them to the map's operational layers collection.
+6. Set the sync direction.
```

Comment:
Between points 5 and 6, should there be a mention of the local edits? (Point 8 talks about the edits)

Reply:
Local edits are now mentioned in step 6. I tried to phrase it while still addressing a developer as the audience. Let me know what you think!

Reply:
Did you push a commit? I don't see it

Reply:
@sbaskaran Sorry, just pushed!